recipes/_linux.rb

```diff
@@ -89,6 +89,31 @@
     flush_cache node['sensu']['yum_flush_cache'] unless node['sensu']['yum_flush_cache'].nil?
     notifies :create, "ruby_block[sensu_service_trigger]"
   end
+when "amazon"
+  repo = yum_repository "sensu" do
+    description "sensu monitoring"
+    repo = node["sensu"]["use_unstable_repo"] ? "yum-unstable" : "yum"
+    releasever_string = node["sensu"]["yum_repo_releasever"] || "7"
```

Comment:
You probably just want to map amazon platform_family of 2.0 to RHEL 7 and then everything else to RHEL 6. Hard mapping to RHEL 7 breaks all the existing Amazon Linux users.